### PR TITLE
upgrade to go 1.13.3 (CVE-2019-17596)

### PR DESCRIPTION
**Pull request by tao12345666333**
full diff https://github.com/golang/go/compare/go1.13.1...go1.13.3

<p>
go1.13.2 (released 2019/10/17) includes security fixes to the
<code>crypto/dsa</code> package and the compiler.
See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.13.2">Go
1.13.2 milestone</a> on our issue tracker for details.
</p>

<p>
go1.13.3 (released 2019/10/17) includes fixes to the go command,
the toolchain, the runtime, <code>syscall</code>, <code>net</code>,
<code>net/http</code>, and <code>crypto/ecdsa</code> packages.
See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.13.3">Go
1.13.3 milestone</a> on our issue tracker for details.
</p>

### CODE REVIEW

**Comment by tao12345666333** [comment]
Wait for Golang docker image has been published. docker-library/official-images#6805

File: hack/go_container.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ SOURCE_DIR="${SOURCE_DIR:-$(pwd -P)}"
 # default to disabling CGO for easier reproducible builds and cross compilation
 export CGO_ENABLED="${CGO_ENABLED:-0}"
 # the container image, by default a recent official golang image
-GOIMAGE="${GOIMAGE:-golang:1.13.1}"
+GOIMAGE="${GOIMAGE:-golang:1.13.3}"
```
Comment (BenTheElder) [inline hack/go_container.sh]: Please also update references in the docs when updating the version here

Comment (tao12345666333) [reply hack/go_container.sh]: I checked the docs before submitting this PR. 
Currently we have used `1.13` directly in our docs without a specific minor version, like this:

>README.md
30:**NOTE**: please use the latest go to do this, ideally go 1.13 or greater.

> site/content/docs/user/quick-start.md
20:**NOTE**: please use the latest Go to do this, ideally go 1.13 or greater.

>site/content/docs/contributing/getting-started.md
54:Preferably Go `1.13` or greater should be installed. 

Should I update all of them to specific minor version?

Comment (BenTheElder) [reply hack/go_container.sh]: hmm you're right, let's leave it for now.

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Comment by tao12345666333** [comment]
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kind/961/pull-kind-build/1185113192976617474/build-log.txt

```
Status: Image is up to date for golang:1.13.3
Digest: sha256:6a693fbaba7dd8d816f6afce049fb92b280c588e0a677c4c8db26645e613fc15
Status: Image is up to date for golang:1.13.3
docker: Error response from daemon: failed to mkdir /docker-graph/volumes/kind-build-cache/_data/bin: mkdir /docker-graph/volumes/kind-build-cache/_data/bin: file exists.
See 'docker run --help'.
make: *** [Makefile:39: kind] Error 125
go: downloading github.com/spf13/cobra v0.0.3
```

[pull-kind-build job](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kind/961/pull-kind-build/1185113192976617474) failed.

**Comment by BenTheElder** [comment]
hmm that is a flake in the cross build

**Comment by tao12345666333** [comment]
emmm I am on vacation.

**Comment by BenTheElder** [comment]
er no worries, it's not a bug in this PR, I'm just circling back to this
and getting it fixed https://github.com/kubernetes-sigs/kind/issues/974

enjoy your vacation!

**Comment by tao12345666333** [comment]
Thank you.

**Pull request merged**